packages/calypso-apps-builder/index.js

```diff
@@ -11,7 +13,7 @@ import { hideBin } from 'yargs/helpers';
 // When we run a build which has more than 10 build tasks in parallel (like ETK),
 // Node shows a warning. We just need to increase the number of allowed listeners
 // to handle larger numbers of parallel tasks.
-EventEmitter.setMaxListeners( 20 );
+EventEmitter.setMaxListeners( 30 );
```

Comment:
This warning is getting triggered because the number of listeners is now above 20. I've increased to 30 for more headroom.